lib/generators/install/templates/index.coffee.erb

```diff
@@ -0,0 +1,8 @@
+#= require cable
+#= require_self
+#= require_tree .
+
+@<%= application_name %> ?= {}
+
+# Define the location of your websockets here
+<%= application_name %>.cable = Cable.createConsumer 'ws://localhost:28080'

```

Comment:
The connection string should be wss-aware: `(window.location.protocol == 'https:' ? 'wss://' : 'ws://')` or something similar. This has caught me out on two deploys.